tests/integration/200.command.dev.test.cjs

```diff
@@ -239,10 +239,15 @@ export const handler = async function () {
 
             if (req.url.includes('?ef=fetch')) {
               const url = new URL('/origin', req.url)
-              const res = await fetch(url)
-              const text = await res.text()
+              try {
+                await fetch(url, {})
+              } catch (error) {
+                return new Response(error)
```

Comment:
nit: I think using an error code here might come in handy.

```suggestion
                return new Response(error, { status: 500 })
```

Reply:
done in 3c9a98c0dd51b778f1cea572b15bee107f2fc6df